### ISSUE TITLE
error messages in ActivityLog.xml

### ISSUE THREAD

**Issue by alanhamlett** [wakatime/visualstudio-wakatime#9]
```
# | Type | Description | GUID | Hr | Source | Time (UTC)
530 | ERROR | CreateInstance failed for package [Main] | {52D9C3FF-C893-408E-95E4-D7484EC7FA47} | 80070002 | VisualStudio | 2015/05/11 14:45:27.475
531 | ERROR | End package load [Main] | {52D9C3FF-C893-408E-95E4-D7484EC7FA47} | 80004005 - E_FAIL | VisualStudio | 2015/05/11 14:45:27.475
```

**Comment by PlatinumEagle** [comment]
Hi Alan,

NOTE: I strongly recommend (it would sure be helpful) that you change the name of the package from "Main" to "WakaTimePackage", which would follow standard conventions for VS Extensions.  Else it is hard to figure out what the heck is the cause, and makes the entries that get put into the registry much harder to associate to the real cause of the problem.
Example: look at the following key in your registry and you'll see the convention for packages and how your package ("Main") shows up: 

    HKEY_CURRENT_USER\Software\Microsoft\VisualStudio\12.0_Config\InstalledProducts
 
Additional information:

	  <entry>
		<record>617</record>
		<time>2015/05/11 23:33:04.510</time>
		<type>Information</type>
		<source>VisualStudio</source>
		<description>Entering function CVsPackageInfo::HrInstantiatePackage</description>
		<guid>{52D9C3FF-C893-408E-95E4-D7484EC7FA47}</guid>
	  </entry>
	  <entry>
		<record>618</record>
		<time>2015/05/11 23:33:04.510</time>
		<type>Information</type>
		<source>VisualStudio</source>
		<description>Begin package load [Main]</description>
		<guid>{52D9C3FF-C893-408E-95E4-D7484EC7FA47}</guid>
	  </entry>
	  <entry>
		<record>619</record>
		<time>2015/05/11 23:33:04.526</time>
		<type>Error</type>
		<source>VisualStudio</source>
		<description>CreateInstance failed for package [Main]</description>
		<guid>{52D9C3FF-C893-408E-95E4-D7484EC7FA47}</guid>
		<hr>80070002</hr>
		<errorinfo>Could not load file or assembly 'Microsoft.VisualStudio.Shell.14.0, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' or one of its dependencies. The system cannot find the file specified.</errorinfo>
	  </entry>
	  <entry>
		<record>620</record>
		<time>2015/05/11 23:33:04.526</time>
		<type>Error</type>
		<source>VisualStudio</source>
		<description>End package load [Main]</description>
		<guid>{52D9C3FF-C893-408E-95E4-D7484EC7FA47}</guid>
		<hr>80004005 - E_FAIL</hr>
		<errorinfo>Could not load file or assembly 'Microsoft.VisualStudio.Shell.14.0, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' or one of its dependencies. The system cannot find the file specified.</errorinfo>
	  </entry>

**Comment by alanhamlett** [comment]
Ok thanks, I've renamed `Main` to `WakaTimePackage`. I also downgraded `Microsoft.VisualStudio.Shell.14.0` to `Microsoft.VisualStudio.Shell.12.0` and released v4.0.2 of the plugin. Can you update WakaTime and see if the error is fixed?

I wasn't able to reproduce the error in my Windows 8 VM. I tried uninstalling all .NET versions, but Visual Studio 2013 and 2015 both still loaded WakaTime correctly without error.

**Comment by PlatinumEagle** [comment]
Confirmed it installs, loads properly and sends events and heartbeats now, both on VS 2013 Ultimate and VS 2015RC Enterprise.

Also, all the registry and logging information is much better now and more easily traceable after the rename, thanks. I'm sure it will help others and you if there are any future issues. I forked the project so that I could take a look, but ran out of time (have a deadline, so I just rebuilt v3.0 and loaded that for a few hours).

-Rob
